### ISSUE TITLE
Index all nodes and properties

### ISSUE THREAD

**Issue by thobalose** [neo4j-contrib/neo4j-elasticsearch#45]
How can I index all nodes and properties using `elasticsearch.index_spec`?

**Comment by jexp** [comment]
Currently you have to spell them out, we'd have to allow something like an "*" for either label or property.

They then would be all get added to the same index?

**Comment by thobalose** [comment]
@jexp 

Yes, that would be great. Spelling the out works well if you have a few nodes.

**Comment by satya06** [comment]
How can we index Node relationship ??
